src/components/Cart/CartListItem.jsx

```diff
@@ -21,7 +21,8 @@ const CartListItem = ({ cartItem }) => {
     const { authState: { token } } = useAuth();
     const { showToast } = useToast();
     const { cartDispatch } = useCart();
-
+    const { callUpdateProductInCart } = useUpdateCart();
+    const { wishListState: { wishListItems }, wishListDispatch } = useWishList();
```

Comment:
if possible you can even destructure this wishlist state in the context itself.

Reply:
This is a good idea. I shall try to do this before submission. If not, do it in temp dev and merge it later to dev.